test/runtests.jl

```diff
@@ -251,6 +251,9 @@ end
 @test sprint(JSON.print, [NaN]) == "[null]"
 @test sprint(JSON.print, [Inf]) == "[null]"
 
+# check for issue #163
+@test isapprox(JSON.parse(json(Float32(2.1e-8))), 2.1e-8)
```

Comment:
ah, good idea. that seems to work on my local tests. Let's make sure CI agrees...